app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,9 @@
+.dashboard_partial{
+  border: 2px solid #003A5B;
```

Comment:
Nothing significant, I just figured I'd use the same colors as in the header because what are we making, a rainbow?